<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="bt_moveleft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::LeftArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="bt_moveup">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::UpArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="bt_movedown">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::DownArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="bt_moveright">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::RightArrow</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_plane">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_stepsize">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stepsize">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step size (mm):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_beam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Beam</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
